src/sentry/notifications/notifications/rules.py

```diff
@@ -157,3 +157,80 @@ def get_log_params(self, recipient: Team | User) -> Mapping[str, Any]:
             "target_identifier": self.target_identifier,
             **super().get_log_params(recipient),
         }
+
+
+class ActiveReleaseAlertNotification(AlertRuleNotification):
+    message_builder = "ActiveReleaseIssueNotificationMessageBuilder"
+    metrics_key = "release_issue_alert"
```

Comment:
Don't know if changing the `metrics_key` to something besides `issue_alert` has any downstream effects.

Reply:
I think this is just used for some analytics, so likely a good thing that it's unique